packages/workspace/src/generators/move/lib/move-project.ts

```diff
@@ -15,11 +15,8 @@ export function moveProject(
 ) {
   const destination = getDestination(tree, schema, project);
   visitNotIgnoredFiles(tree, project.root, (file) => {
-    // This is a rename but Angular Devkit isn't capable of writing to a file after it's renamed so this is a workaround
-    const content = tree.read(file);
```

Comment:
I think it's safer to revert this change, the Angular DevKit doesn't support yet to rename and update a file, this workaround still applies.

Reply:
Ok. I had made the change as I had assumed that this would only ever be used by Nx Devkit but I guess someone can still run this as a schematic